### PR TITLE
[Documentation]

### PR DESCRIPTION
**Pull request by AghaSaad04**
Signed-off-by: Agha Saad Fraz agha.saad04@gmail.com
I have to document this library while working as a technical writer for Arduino in GSoD. I have documented the DHT-sensor library as per David Cuartielles' plan.

I have done the following steps

Followed writing standards
Added Description
Added Installation steps
Added Features
Added functions
Added example
Added contributing heading and its file
Added credits
Referenced external resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,166 @@
...
+
+## Credits
+
+The author and maintainer of this library is Adafruit <info@adafruit.com>
+
+Based on previous work by:
+
+- T. DiCola
+- P. Y. Dragon
+- L. L. Fried
```
Comment (ladyada) [inline README.md]: No middle name, just L. Fried

File: README.md [hunk 2]
```diff
@@ -1,14 +1,166 @@
...
+- J. Lim
+- G. Ambrozio
+- chelmi
+- adams13x13
+- Spacefish
+- I. Scheller
+- C. Miller
+- 7eggert
+
+## Current stable version
```
Comment (ladyada) [inline README.md]: please remove this section, we will not be updating it. they can check the releases tab for latest version number

File: README.md (hunk 1 above)
Comment (AghaSaad04) [reply README.md]: Thanks, @ladyada  for reviewing. I have just fixed this one :)

File: README.md (hunk 2 above)
Comment (AghaSaad04) [reply README.md]: I removed this section.

**Pull request merged**